hybridse/src/udf/default_defs/feature_zero_def.cc

```diff
@@ -379,6 +379,11 @@ struct FZStringOpsDef {
         output->size_ = bytes;
         output->data_ = buf;
     }
+
+    static void ListSize(hybridse::codec::ListRef<StringRef>* list, int32_t* size) {
+        auto list_v = reinterpret_cast<hybridse::codec::ListV<StringRef> *>(list->list);
+        *size = list_v->GetCount();
```

Comment:
if the result value is not-null,  return by the function return value? refer [build-in function docs] (https://github.com/4paradigm/OpenMLDB/blob/main/docs/zh/developer/built_in_function_develop_guide.md?plain=1#L91)